### PR TITLE
8.0 partner salutation therp

### PR DESCRIPTION
**Pull request by NL66278**
Add partner salutation module. This module gives the user complete control over the salutation of the customers in letters and reports.

### CODE REVIEW

**Comment by hbrunn** [comment]
would it make sense to move `res_partner_title.salutation` from `l10n_partner_name_dutch` to this module? It was more a matter of convenience back then to have it in `l10n_partner_name_dutch`, but given that now a salutation module exists, I think that's the better place to put it.

**Comment by NL66278** [comment]
@hbrunn  I moved over everyting to do with salutation to the appropiate module.

**Comment by hbrunn** [comment]
great, thanks! :+1:

**Comment by diellenne** [comment]
Thank you for your contribution; I'm going to review it during this week.
Just a general thing: next time you want to propose a new module to be added to the Dutch localization project will be nice to talk about it before you made the pull request. Just a simple post on the issue system will be enough...

Thanks again,
D

**Comment by hbrunn** [comment]
@diellenne you seem to misapprehend your role in this project. You are one reviewer of many, and whoever does the third (or second, see below) positive review, can merge the code in question: https://github.com/OCA/maintainer-tools/wiki/Conventions#review

And I think the good moment to talk about a contribution is when it is available, which would be the PR

File: l10n_nl_partner_salutation/README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+While organizations have no gender (the field is hidden there), you can have
+a feminine salutation by creating a partner title with an implicit gender,
+such as 'Ladies club'.
+
+Contributors
+------------
+
+* Holger Brunn <hbrunn@therp.nl>
+* Ronald Portier <ronald@therp.nl>
```
Comment (StefanRijnhart) [inline l10n_nl_partner_salutation/README.rst:27]: As the original author of the module, can I have a mention ;-)

**Comment by diellenne** [comment]
Dear @hbrunn 
as you I am a reviewer of many, but I'm also the leader of the project...for this reason is one of my duties to try to keep the development aligned and try to define, with all of you, a road-map to work together in a more effective way. For this reason I think that when somebody wants to start a new module related to the Dutch Localization, for sure he doesn't have to ask the permission, but will be nice to share his ideas with the others; in this way everybody can know that you are working on something and maybe help you.

Gr,
D

**Comment by StefanRijnhart** [comment]
For the record: the word 'project leader' is a leftover from the notion
in Odoo itself, reflecting where this information is stored in the OCA
instance. It does not indicate special privileges compared to other
contributors to the project. To avoid the confusion, the OCA now has
changed all occurrences of 'Project leader' to 'OCA representatives' on
its website. See https://odoo-community.org/page/suggest-a-team for
instance.

**Comment by diellenne** [comment]
Ok, thank you for the update.
By the way this doesn't change that I still believe that is good to inform the community if you are going to work on a new community module.

**Comment by lfreeke** [comment]
:+1: Test

**Comment by StefanRijnhart** [comment]
This PR also contains the changes in #15. Can you clarify?

**Comment by NL66278** [comment]
If approving #16 PR #15 will no longer be relevant, but #15 contains a simple migration to 8.0 of l10n_nl_partner_name. Which is required for l10n_nl_partner_salutation.

**Comment by StefanRijnhart** [comment]
So if #15 is included in this pull request, you asked us to review the same code twice. The remark I made in this PR then actually applies to #15. Maybe we can simply close #15 now? Note that I do appreciate piecemeal PRs, but I take care myself that I then propose follow up PRs only when the prerequisites have been merged.

**Comment by NL66278** [comment]
OK, I closed #15

**Comment by hbrunn** [comment]
@NL66278 could you include @StefanRijnhart as a contributor please so that I can go on with the merge?

**Comment by hbrunn** [comment]
@NL66278 I just tested the module with current partner_firstname, and it seems the compute method changed: https://github.com/OCA/partner-contact/blob/8.0/partner_firstname/models.py#L43
So at the moment, you only get firstname + lastname from the partner_firstname module

**Comment by andreschenkels** [comment]
@hbrunn @StefanRijnhart is it a good idea to redo this part or are you working on this? Should the name of the module be changed? You removed the salutation part from the module isn't it?

**Comment by hbrunn** [comment]
@aschenkels-ictstudio the salutation part is its own module, contained in the PR as well. To get stuff on track: Clone this branch, rebase on current 8.0, add @StefanRijnhart, then it's ready to merge

**Comment by hbrunn** [comment]
@NL66278  @lfreeke @StefanRijnhart @aschenkels-ictstudio I close this in favor of #26

**Pull request closed without merging**